### PR TITLE
chore: Update NavigateTo tests to verify params

### PR DESCRIPTION
**Pull request by hetunandu**
## Description

Recently, navigation broke where params were not retained when using the navigateTo function. Adding an assertion here to make sure it does not break again

### CODE REVIEW

File: app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/ActionExecution/NavigateTo_spec.ts [hunk 1]
```diff
@@ -29,9 +28,16 @@ describe("Navigate To feature", () => {
     cy.get(".t--open-dropdown-Select-Page").click();
     agHelper.AssertElementLength(".bp3-menu-item", 2);
     cy.get(locator._dropDownValue("Page2")).click();
+    cy.get("label")
+      .contains("Query Params")
+      .siblings()
+      .find(".CodeEditorTarget")
+      .then(($el) => cy.updateCodeInput($el, "{{ { test: '123' } }}"));
     deployMode.DeployApp();
```
Comment (rahulramesha) [inline app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/ActionExecution/NavigateTo_spec.ts:36]: @hetunandu the issue was only in edit mode and this test in view mode, which did not break. Also, i remember @Aishwarya-U-R was also working on a similar spec.

**Review by rahulramesha — CHANGES_REQUESTED** [review]

File: app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/ActionExecution/NavigateTo_spec.ts (hunk 1 above)
Comment (hetunandu) [reply app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/ActionExecution/NavigateTo_spec.ts:40]: Updated to add edit mode assertion

**Review by rahulramesha — APPROVED** [review]

**Review by akash-codemonk — APPROVED** [review]

**Pull request merged**